### ISSUE TITLE
V-Select Not Showing Label If Value is Zero 

### ISSUE THREAD

**Issue by Marcellofabrizio** [sagalbot/vue-select#1079]
I'm using the `<v-select>` component with a set of keys and values. The data I'm showing is:
` data: [ { key: 0, value: Residential }\ { key: 1, value: Comercial }\ { key: 2, value: Personal }] ` 

  And the component looks like this:
` <v-select>\ placeholder= "Marker"\ label="value"\ :options="data"\ :reduce="marker => marker.key\ v-model="data.marker"\ :value="data.marker"\  `

  Now, if `key` is zero, `v-select` won't show the `value` after I reload the page, as if it was null, only showing the placeholder. I'm using V-Select version 3.5.1

**Comment by sagalbot** [comment]
Will need a codepen or jsbin reproduction link to investigate. If you can provide a link, I'm happy to look into it.

**Comment by caryxiao** [comment]
```js
options=[
  { label: 0, value:0 }
  { label: 1, value: 1 }
]

vue-select set v-model=0

vue-select default show selectd text is empty. 
```

![image](https://user-images.githubusercontent.com/2449891/78035478-bdd1f480-739b-11ea-8235-a5f59ec32dbf.png)

**Comment by kerimkuscu** [comment]
@sagalbot  Yes, I am having the same problem (null) and (int 0) when the value is selected, it does not appear on the screen.

**Comment by kerimkuscu** [comment]
@sagalbot ?

**Comment by nicholasscottfish** [comment]
@sagalbot @kerimkuscu Did anybody find a solution to this?

**Comment by Marcellofabrizio** [comment]
@nicholasscottfish Still on the dark about this

**Comment by kerimkuscu** [comment]
@nicholasscottfish I don't use this library anymore.

**Comment by nicholasscottfish** [comment]
@Marcellofabrizio @kerimkuscu 

Just in case you do ever need this, I have come up with a workaround.

In my particular case I was originally passing in the values as a on dimensional array as so:

[0,1,2,3,4,5,6]

However, I found that if I actually pass in the values as an array of objects and use the reduce prop, this would actually display the 0 value:

`
[
{value: 0},
{value: 1},
...
{value: 6}
]
`

For the v-select component you then specify that you want to use the value as the label and use the reduce to set the option value (adultCoverOptions being in the above format:

`
<v-select
    v-model="displayAdultCovers"
    label="value"
    :clearable="false"
    :options="adultCoverOptions"
    :reduce="option => option.value"
    :searchable="false"
    placeholder="How many adults?*"
 />
`

There was a knock-on effect with doing this however, the 0 would be set by default when binding via v-model and so I used this workaround found here to split the data attribute into a "display" version and the actual value I need.

https://stackoverflow.com/questions/43320196/transform-v-model-bound-value-in-vue-js

If you have any questions on any of the above just let me know and I'll be happy to help out where I can.
